cloudnet_api_client/client.py

```diff
@@ -224,6 +252,43 @@ def raw_model_metadata(
         res = self._get_response("raw-model-files", params)
         return _build_raw_model_meta_objects(res)
 
+    def moving_site_mean_location(
+        self, site_id: str, date: datetime.date | str
+    ) -> Location:
+        dd = _parse_date_param(date)[0]
```

Comment:
Ei ole "date param" vaan "date":

```suggestion
        if not instanceof(date, datetime.date):
            date = datetime.date.fromisoformat(date)
```

Reply:
Hups, joo totta...